tests/terraform/e2e_controller/main.tf

```diff
@@ -0,0 +1,78 @@
+data "aws_ami" "latest_controller" {
+  most_recent = true
+  owners      = ["600530653622"]
+
+  filter {
+    name   = "name"
+    values = ["avx-controller-g4-*"]
+  }
+}
+
+# try to satisfy the controller's password strength meter by combining
+# two passwords with a special char
```

Comment:
Can this fail sometimes to meet that requirement?  The docstring makes it it sound like maybe?

Reply:
I have not seen it fail. This was initially written by Ross/Joe for their testing and has gone through a couple of iterations, and it seems to work well. I can remove the comment if it's confusing